R/ebola_linelist_2014_doc.R

```diff
@@ -0,0 +1,22 @@
+#' @title Fictional Ebola linelist data from fSierra Leone, 2014
+#'
+#' @description
+#' Fictional linelist data based on the real emergence of Ebola in West Africa in 2014. 
+#'  
+#' @details
+#' This is a case linelist which contains fictional individual level data from one country
+#'  on clinical, laboratory and epidemiological information.
+#' It is intended for use as part of Applied Epi training courses, 
+#' alongside \link{mpox_aggregate_table} (cumulative case totals by date and country). 
+#' 
+#'  
+#' @source Fictional data provided by Applied Epi for training purposes
```

Comment:
the original is from the {outbreaks} package and was edited, so would need to be acknowledged here.